### PR TITLE
New package: qt-virt-manager-0.72.98

### PR DESCRIPTION
**Pull request by Logarithmus**
Qt Virtual machines manager

### CODE REVIEW

**Comment by Logarithmus** [comment]
@ericonr @pullmoll @Duncaen i think it can be merged now

File: srcpkgs/qt-virt-manager/template [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Template file for 'qt-virt-manager'
+pkgname=qt-virt-manager
+version=0.71.95
+revision=1
+build_style=cmake
+hostmakedepends="pkg-config"
+makedepends="qt5-multimedia-devel qt5-svg-devel qtermwidget-devel \
+	libglib-devel spice-gtk-devel libvncserver-devel \
```
Comment (Duncaen) [inline srcpkgs/qt-virt-manager/template:8]: Indention with just one space and \ at the end of lines inside quotes variables is not required.

File: srcpkgs/qt-virt-manager/template [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+	libvirt-devel krdc-devel qt5-tools-devel"
+depends="qt5-xml"
+short_desc="Qt Virtual machines manager"
+maintainer="Artur Sinila <opensource@logarithmus.dev>"
+license="GPL-2.0-only"
+homepage="https://f1ash.github.io/qt-virt-manager/"
```
Comment (Duncaen) [inline srcpkgs/qt-virt-manager/template:14]: Add
```
changelog="https://github.com/F1ash/qt-virt-manager/raw/master/Changelog"
```

**Comment by Duncaen** [comment]
Please don't ping me if I'm not strictly involved, this just adds noise to my notifications and I generally review or merge PRs on my own accord.

**Comment by Logarithmus** [comment]
I'm sorry for that. I've added the changelog and corrected the spaces.

**Comment by Logarithmus** [comment]
@ericonr ping?

**Review by zen0bit — APPROVED** [review]
title should be (I think): New package: qt-virt-manager 0.72.98
otherwise
:+1:

I didn't even known that exist and I like it...

Thanks @Logarithmus

**Comment by zen0bit** [comment]
new version 0.72.99

File: srcpkgs/qt-virt-manager/template [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# Template file for 'qt-virt-manager'
+pkgname=qt-virt-manager
+version=0.72.98
```
Comment (zen0bit) [inline srcpkgs/qt-virt-manager/template:3]: version=0.72.99

File: srcpkgs/qt-virt-manager/template [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+ libglib-devel spice-gtk-devel libvncserver-devel
+ libvirt-devel krdc-devel qt5-tools-devel"
+depends="qt5-xml"
+short_desc="Qt Virtual machines manager"
+maintainer="Artur Sinila <freesoftware@logarithmus.dev>"
+license="GPL-2.0-only"
+homepage="https://f1ash.github.io/qt-virt-manager/"
+changelog="https://github.com/F1ash/qt-virt-manager/raw/master/Changelog"
+distfiles="https://github.com/F1ash/qt-virt-manager/archive/${version}.tar.gz"
+checksum=bf8b0ffe08453224efcb60a3a9f49a60f0fa195a345f996463540c78d523ec91
```
Comment (zen0bit) [inline srcpkgs/qt-virt-manager/template:17]: checksum=06b451d32ea427c17a5520f26adfc49848ea348a8fa3b6cf73d74813e6cd0b5f

**Review by zen0bit — CHANGES_REQUESTED** [review]
New version 0.72.99 is out...

File: srcpkgs/qt-virt-manager/template [hunk 5]
```diff
@@ -0,0 +1,21 @@
+# Template file for 'qt-virt-manager'
+pkgname=qt-virt-manager
+version=0.72.98
+revision=1
+build_style=cmake
+hostmakedepends="pkg-config"
+makedepends="qt5-multimedia-devel qt5-svg-devel qtermwidget-devel
+ libglib-devel spice-gtk-devel libvncserver-devel
+ libvirt-devel krdc-devel qt5-tools-devel"
+depends="qt5-xml"
```
Comment (zen0bit) [inline srcpkgs/qt-virt-manager/template:10]: Looks like this also needed as dependency:
krdc qtermwidget

**Review by zen0bit — CHANGES_REQUESTED** [review]

**Pull request closed without merging**